src/dso_api/dynamic_api/remote/views.py

```diff
@@ -153,7 +153,7 @@ def _call_remote(self, url="", query_params={}) -> Union[dict, list]:  # noqa: C
         if query_params:
             url = urljoin(self.endpoint_url, "?" + urlencode(query_params))
 
-        url = url.replace("{table_id}", self.table_id)
+        url = url.format(table_id=self.table_id)
```

Comment:
Ik denk dat @bheupers dit expres zo gedaan heeft: er worden nu geen geavanceerde formatting rules ondersteund. Alleen de vervanger van een placeholder die toevallig python-formatting syntax gebruikt.

Reply:
Het voordeel hiervan is dat je een exception krijgt als {table_id} niet voorkomt in de URL. Maar daar zou ik natuurlijk een check voor kunnen schrijven. Nadeel is dat accolades in URLs verdubbeld moeten worden, maar die zijn volgens mij zeldzaam.